config/initializers/edition_services.rb

```diff
@@ -14,6 +14,9 @@
       ServiceListeners::AttachmentRedirectUrlUpdater
         .new(attachment)
         .update!
+      ServiceListeners::AttachmentLinkHeaderUpdater
+        .new(attachment)
+        .update!
```

Comment:
Minor: I wonder whether it might be worth being more selective about the events we do this for? However, this is potentially a problem for other updaters, so perhaps we should address it separately.

Reply:
I've changed this in another commit in this branch. I'm now only subscribing to the `publish` and `force_publish` events.